XenOvfTransport/Import.cs

```diff
@@ -1226,109 +1226,67 @@ private XenRef<VM> DefineSystem(Session xenSession, VirtualHardwareSection_Type
             }
             #endregion
 
+            var longMemorySize = memorySize > long.MaxValue ? 0 : (long)memorySize;

```

Comment:
Why is the `longMemorySize `set to 0 if is greater than `long.MaxValue`? Why not `long.MaxValue`?

Reply:
I was wondering about that too, but I'm not sure what the correct solution is. The max value seems to make more sense, but I left it 0 because this is what the Marshalling.ParseLong function also does.

Reply:
Ok then.
Speaking of the Marshalling class., there are a lot of Parse functions there that are not used anymore after these changes. Shall we remove them?

Reply:
Unfortunately they are public :( I think this was a design bug as I doubt there was the intention to use them externally, but since it happened, we should first deprecate them for a little while.